include/net/nrf_cloud_cell_pos.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_CLOUD_CELL_POS_H_
+#define NRF_CLOUD_CELL_POS_H_
+
+/** @file nrf_cloud_cell_pos.h
+ * @brief Module to provide nRF Cloud cellular positioning support to nRF9160 SiP.
+ */
+
+#include <zephyr.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @defgroup nrf_cloud_cell_pos nRF Cloud cellular positioning
+ * @{
+ */
+
+/** @brief Cell-based location request type */
+enum cell_based_location_type {
+	CELL_LOC_TYPE_INVALID = -1,
+	CELL_LOC_TYPE_SINGLE,
+	CELL_LOC_TYPE_MULTI /* Not yet supported */
+};
+
+/**@brief Request a cell-based location query from nRF Cloud.
+ *
+ * @param type        Type of cell-based location to request.
+ * @param request_loc If true, cloud will send location to the device.
+ *                    If false, cloud will not send location to the device.
+ * @return 0 if successful, otherwise a (negative) error code.
+ */
+int nrf_cloud_request_cell_location(enum cell_based_location_type type, const bool request_loc);
+
+/**@brief Gets most recent location from cellular positioning request.
+ *
+ * @param lat Pointer where last cellular positioning latitude is to be copied.
+ * @param lon Pointer where last cellular positioning longitude is to be copied.
+ * @return 0 if successful, otherwise a (negative) error code.
+ */
+int nrf_cloud_get_last_cell_location(double *const lat, double *const lon);
+
+/**@brief Processes cellular positioning data received from nRF Cloud.
+ *
+ * @param buf Pointer to data received from nRF Cloud.
+ *
+ * @return 1 if processed successfully and cell-based location found.
+ *         0 if processed successfully but no cell-based location found.
+ *         otherwise a (negative) error code.
```

Comment:
I think it would be better to return 0 for the success case where location is found and 1 when no location is found. This way zero would mean success and any other value an error (it's also a kind of an error that no location was found).

Reply:
See Justin's comment above.

Reply:
Where? :)

Reply:
it was marked as resolved:
https://github.com/nrfconnect/sdk-nrf/pull/4340#discussion_r640843526

the return value of 1 was added but the description is not correct as tommi's mentions